### DISCUSSION TITLE
Hosted renovate interferes with pull requests raised by self-hosted renovate

### DISCUSSION THREAD

**Discussion by ChipWolf** [renovatebot/renovate#14013] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

![](https://i.cwlf.uk/kEo5O)

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by ChipWolf** [comment]
Oddly, `ignorePrAuthor` is false

**Comment by viceice** [comment, marked as answer]
you need to use different branch prefix

https://docs.renovatebot.com/configuration-options/#branchprefix

**Reply by ChipWolf** [reply]
What is ignorePrAuthor for in this case?

**Reply by rarkins** [reply]
For when you change PR authors and need to migrate
